### PR TITLE
[VEN-1731]: deploy HAY and SD rewards distributor

### PR DESCRIPTION
**Pull request by kkirka**
This PR features new HAY rewards distributor and modifies how rewards distributors' deployment names are generated

### CODE REVIEW

**Comment by chechu** [comment]
Just I reminder: we should merge develop to main after merging this PR, to release a new public version of the npm package

File: helpers/deploymentConfig.ts [hunk 1]
```diff
@@ -1066,6 +1072,12 @@ export const globalConfig: NetworkConfig = {
             supplySpeeds: ["3703703703703703"], // 3200 SD over 30 days (864000 blocks)
             borrowSpeeds: ["3703703703703703"], // 3200 SD over 30 days (864000 blocks)
           },
+          {
+            asset: "SD",
+            markets: ["BNBx"],
+            supplySpeeds: ["1157407407407407"], // 1000 SD over 28 days (806400 blocks)
+            borrowSpeeds: ["1157407407407407"], // 1000 SD over 28 days (806400 blocks)
```
Comment (GitGuru7) [inline helpers/deploymentConfig.ts:1079]:
```suggestion
            supplySpeeds: ["1240079365079365"], // 1000 SD over 28 days (806400 blocks)
            borrowSpeeds: ["1240079365079365"], // 1000 SD over 28 days (806400 blocks)
```

File: helpers/deploymentConfig.ts [hunk 2]
```diff
@@ -1759,6 +1777,12 @@ export const globalConfig: NetworkConfig = {
...
```
Comment (GitGuru7) [inline helpers/deploymentConfig.ts:1784]:
```suggestion
            supplySpeeds: ["1240079365079365"], // 1000 SD over 28 days (806400 blocks)
            borrowSpeeds: ["1240079365079365"], // 1000 SD over 28 days (806400 blocks)
```

File: helpers/deploymentConfig.ts (hunk 1 above)
Comment (chechu) [reply helpers/deploymentConfig.ts:1079]: The value is correct, the comment is wrong. I'll fix it

File: helpers/deploymentConfig.ts (hunk 2 above)
Comment (chechu) [reply helpers/deploymentConfig.ts:1784]: The value is correct, the comment is wrong. I'll fix it

**Review by Debugger022 — APPROVED** [review]

**Pull request merged**